Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -632,6 +633,10 @@ void NativeReanimatedModule::performOperations() {
               const ShadowNodeFamily &family = shadowNode->getFamily();
               react_native_assert(family.getSurfaceId() == surfaceId_);
 
+              if (propsRegistry_->shouldSkipCommit()) {
+                return nullptr;
+              }
+
```

Comment:
I'm not particularly happy that we read the atomic for each animated component but I guess it's not that expensive compared to cloning the whole ShadowTree which is built from `ShadowNode::Shared`.

Reply:
I guess so, do we have performance tests that can measure impact of this change?

Reply:
Unfortunately we don't have setup for perftests yet.